### ISSUE TITLE
Parse Error - RS2 and RS3 must be clear

### ISSUE THREAD

**Issue by kyletan7** [jaggedsoft/node-binance-api#74]
I opened a thread [https://github.com/jaggedsoft/node-binance-api/issues/60](here) about this a couple of days ago. It got closed, so I thought it was fixed, but had the same error again today. Same method  used as described in that case. Here's what I got from the log:

```
Parse error: Unexpected token n in JSON at position 59659 
error: uncaughtException: RSV2 and RSV3 must be clear 
```
I'm thinking that Binance ocasionally sends corrupt JSON over\some illegal character somewhere which causes the error. The problem is, although rare, it crashes the entire application immediately if encountered  since there is no way to catch/detect the error in the callbacks.

**Comment by jaggedsoft** [comment]
I can't seem to reproduce this, so I'm not sure how to fix it. Sorry
But I am open to suggestions

**Comment by kyletan7** [comment]
It seems to be a Binance issue
https://stackoverflow.com/questions/45303733/error-rsv2-and-rsv3-must-be-clear-in-ws

**Comment by jaggedsoft** [comment]
Hopefully this is fixed by the next release `v0.4.8` otherwise let me know
We are now handling unexpected websocket responses
